blockchain/types/accountkey/doc.go

```diff
@@ -14,5 +14,38 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package accountkey contains types related to keys in Klaytn accounts.
+/*
+The package accountkey implements the AccountKey used in Klaytn.
+Inside the package are the types, functions and interfaces associated with the AccountKey.
+
+Type of AccountKey
+
+The AccountKey types used in Klaytn are as follows:
+  - AccountKeyTypeNil
+  - AccountKeyTypeLegacy
+  - AccountKeyTypePublic
+  - AccountKeyTypeFail
+  - AccountKeyTypeWeightedMultiSig
+  - AccountKeyTypeRoleBased
+
+Each AccountKey type implements the AccountKey interface.
+
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey type and the AccountKey interface, and defines the functions related to AccountKey.
+  - account_key_fail.go               : AccountKeyFail is defined according to the AccountKey interface.
+  - account_key_legacy.go             : AccountKeyLegacy is defined according to the AccountKey interface.
```

Comment:
How about write like this? Just my opinion. :)
`An AccountKey for legacy key is defined.`
`An AccountKey for nil key is defined.`
Also, if legacy key, nil key .. can be explained, it is better.

Reply:
Description is edited ! :) Please take a look !